Svc/SeqDispatcher/SeqDispatcherCommands.fppi

```diff
@@ -0,0 +1,9 @@
+async command RUN(
+                      fileName: string size 240 @< The name of the sequence file
+                      $block: Svc.CmdSequencer.BlockState @< Return command status when complete or not
+                    ) \
+    opcode 0
+
+async command LOG_STATUS() opcode 1
+
+async command PING() opcode 2
```

Comment:
What is PING for?  Typically, we do ping ports as those can be tracked by health.

Reply:
Hmm, internally we add them so that we can ping the component's thread. How do you hook it up to health?

Reply:
Removed ping command